### PR TITLE
mdlint and vale editorial updates - index.md

### PR DESCRIPTION
**Pull request by grantnoble**
Line length <= 100 in anticipation of an mdlint customized style file.
Removed trailing spaces.
Some minor wording updates.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
-The Besu client supports common smart contract and Dapp development, deployment, and operational use cases, using tools such as [Truffle](http://truffleframework.com/), [Remix](https://github.com/ethereum/remix), and [web3j](https://web3j.io/). The client supports common JSON-RPC API methods such as eth, net, web3, debug, and miner.
+The Besu client supports common smart contract and Dapp development, deployment, and operational
+use cases, using tools such as [Truffle](http://truffleframework.com/),
+[Remix](https://github.com/ethereum/remix), and [web3j](https://web3j.io/). The client supports
+common JSON-RPC API methods such as eth, net, web3, debug, and miner.
 
-Besu doesn't support [key management](HowTo/Send-Transactions/Account-Management.md) inside the client. You can use 
-[EthSigner](http://docs.ethsigner.pegasys.tech/en/latest/) with Besu to provide access to your key store
-and sign transactions.  
+Besu does not support [key management](HowTo/Send-Transactions/Account-Management.md) inside the
```
Comment (NicolasMassart) [inline docs/index.md]: MS guidelines (that we use) say to use contractions : https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions

File: docs/index.md [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
-over HTTP or via WebSockets, and Pub/Sub is supported. The API supports typical Ethereum functionalities such as:
+Besu includes a [command line interface](Reference/CLI/CLI-Syntax.md) and
+[JSON-RPC API](HowTo/Interact/APIs/API.md) for running, maintaining, debugging, and monitoring
+nodes in an Ethereum network. You can use the API via RPC over HTTP or via WebSockets. Besu also
+supports Pub/Sub. The API supports typical Ethereum functionalities such as:
 
 * Ether mining
 * Smart contract development
-* Decentralized application (Dapp) development
+* Decentralized application (Dapp) development.
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/index.md]: Period at the end of a bulleted list to indicate the end of the sentence.

File: docs/index.md (hunk 1 above)
Comment (grantnoble) [reply docs/index.md]: OK.

File: docs/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (grantnoble) [reply docs/index.md]: Personally, I'm not a fan of ; at the end of bullet points. It depends on our style guide, but I advocate periods at the end of each point (if each point is its own sentence) or a period at the end of the list of items.

[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/index.md]: ; is against our style guide (I'm with you @grantnoble on that one) and yep, period on the end of each point if they're standalone or on the last one if they're a list.

File: docs/index.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/index.md]: @grantnoble - contractions took me a while to get used to!

**Pull request merged**